src/libraries/Microsoft.PowerFx.Interpreter/EvalVisitor.cs

```diff
@@ -341,6 +341,30 @@ public override FormulaValue Visit(RecordFieldAccessNode node, SymbolContext con
                 return left;
             }
 
+            if (left is CustomObjectValue cov)
```

Comment:
> if (left is CustomObjectValue cov) [](http://example.com/codeflow?start=11&length=35)

Design question here....

We want to make an IR easy to implement, and easy to know what's *not* implemented - because multiple backends are handling. 
Binary operator has an enum, and so that's easy to know which kinds are nyi. So the IR nodes should be translated enough that we can avoid switching logic like this (how does a backend implementor know the various cases?)

Along that, what do you think of translating Co.field as a binary operator with a new BinaryOpKind.DynamicGetField?  
If we add GetField() function, that can also translate to this op. (I think And/Or have a similar issue) 

(@lesaltzm fyi)